### ISSUE TITLE
Code Table Request - window strike

### ISSUE THREAD

**Issue by wellerjes** [ArctosDB/arctos#7795]
### Goal
_Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step._
record how many birds in our collection (CHAS:Bird) have been killed due to window strike

### Context
_Describe why this new value is necessary and existing values are not._
This is something we report on and "mortality caused" is a new code table

### Table
_Code Tables are http://arctos.database.museum/info/ctDocumentation.cfm. Link to the specific table or value. This may involve multiple tables and will control datatype for Attributes. OtherID requests require BaseURL (and example) or explanation. Please ask for assistance if unsure._
https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmortality_cause

### Proposed Value
_Proposed new value. This should be clear and compatible with similar values in the relevant table and across Arctos._
window strike

### Proposed Definition
_Clear, complete, non-collection-type-specific **functional** definition of the value. Avoid discipline-specific terminology if possible, include parenthetically if unavoidable._
an animal that dies through impact with a window; most likely birds that cannot see the glass due to reflection of sky and foliage

### Collection type
_Some code tables contain collection-type-specific values. ``collection_cde`` may be found from https://arctos.database.museum/home.cfm_
Bird, Teach

#### Attribute data type
_If the request is for an attribute, what values will be allowed?
free-text, categorical, or number+units depending upon the attribute (TBA)_
window strike

#### Attribute controlled values
_If the values are categorical (to be controlled by a code table), add a link to the appropriate code table. If a new table or set of values is needed, please elaborate._
https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmortality_cause

#### Attribute units
_if numerical values should be accompanied by units, provide a link to the appropriate units table._
n/a

#### Part preservation attribute affect on "tissueness"
_if a new part preservation is requested, please add the affect it would have on "tissueness": No Influence, Allows, or Denies_
n/a

### Priority
_Please describe the urgency and/or choose a priority-label to the right. You should expect a response within two working days, and may utilize [Arctos Contacts](https://arctosdb.org/contacts/) if you feel response is lacking._

### Example Data
_Requests with clarifying sample data are generally much easier to understand and prioritize. Please attach or link to any representative data, in any form or format, which might help clarify the request._

### Available for Public View
_Most data are by default publicly available. Describe any necessary access restrictions._
n/a

### Helpful Actions

- [ ] Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184).

- [ ] Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary.

@ArctosDB/arctos-code-table-administrators 

## Approval
_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [ ] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection
_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation
_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

- [ ] Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel.

- [ ] Add or revise the code table term/definition as described above. Ensure the URL of this Issue is included in the definition. URLs should be included *_as text_*, separated by spaced pipes. Do not include HTML in definitions.

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by campmlc** [comment]
I support this. Actively used data.

**Comment by dustymc** [comment]
Is this level of specificity necessary? I'm not sure a list of everything that anything's ever failed to survive crashing into can work here! (And I think there's some possibly some method involved in 'window'?)

**Comment by campmlc** [comment]
These are tracked and recorded as "window strike" by ornithologists all over the world. It's not hit by car or struck by a stick or a meteor.

**Comment by wellerjes** [comment]
This is a specific subset of data, numbers of which are reported to Fish and Wildlife Services. There are collision monitoring programs all over to track how migratory birds are impacted by buildings in residential areas. Monitoring change in time of window strikes helps determine if the changes implemented by these programs are working. Lumping in window strikes with vehicular strikes will take more time/effort to sort through the data for reporting.

https://www.fws.gov/sites/default/files/documents/reducing-bird-collisions-with-buildings.pdf

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
This seems good to me. I will check a box.

> Is this level of specificity necessary?

Frequency of use for this value and reporting requirements seem like great reasons for including this.

**Comment by campmlc** [comment]
I checked a box. Here's the term and some explanation from Cornell Lab of Ornithology:
https://www.allaboutbirds.org/news/why-birds-hit-windows-and-how-you-can-help-prevent-it/#

**Comment by dustymc** [comment]
<img width="181" alt="Screenshot 2024-05-17 at 15 28 43" src="https://github.com/ArctosDB/arctos/assets/5720791/3c0f89a3-ebb8-4ee7-93d0-5524408ae63a">
<img width="170" alt="Screenshot 2024-05-17 at 15 29 30" src="https://github.com/ArctosDB/arctos/assets/5720791/b3774bbc-4076-4dac-b45f-2da3b5aec061">

Birds hit other parts of buildings (and other structures, but I'd guess that towers and generators and such are their own thing) for lots of reasons, and glass != window. Are windows treated differently than other parts of buildings?

**Comment by campmlc** [comment]
Yes. There are a lot of tracking and mitigation efforts specifically for window strikes/collisions, which differ from vehicles or wind turbine etc. The vast majority of strike deaths over a billion in North America, are specifically with windows because the birds can't see the glass or see reflection of open country.

**Comment by acdoll** [comment]
People will be searching on 'window' to find these types of records. Seems like this is exactly the type of thing this table is created for.

**Comment by mkoo** [comment]
If we're going here then makes sense to add 'dead on road" and get that out of remarks. Also tagged and tracked by several organizations. Let me know when the CT gets created so we can file a new issue

**Comment by dustymc** [comment]
>searching

There's a mechanism to support search by alternate terms in this code table.

>dead on road"

The CT has been created (and is linked in the initial comment).

FWIW DOR will make me ask the same kinds of questions - do we really mean ON the road, and if not then why confuse things by using that term (common though it may be)? "Vehicle" or something of the sort would be more correct, no?  (And I still don't know if we're trying to say "just windows" here, or if that is a term meant to include buildings or glass or ?????)

**Comment by cjconroy** [comment]
I know we have put DOR in various places, but I think having it in event remark is simply information about how the specimen was acquired. It makes no assumption about how it was killed. And it makes it clear the specimen was not trapped, shot, euthanized, etc.

**Comment by campmlc** [comment]
These are standard terms that have been in use by biologists for decades. I think that making them as simple as possible to find without overthinking is the best way to go here.

**Comment by mkoo** [comment]
Now that the table exists, there seems to be no db structural issues preventing this new request. For the sake of data entry, I propose we go ahead with "window strike". Synonyms can be added in the future as well as refinement the definition

**Comment by mkoo** [comment]
I see that a one more db check is needed to proceed. I am guessing the issue is of a semantic nature. I am guessing that may stem from the CT name itself. This really should be not a CT for "mortality caused" but "mortality condition" or "situation". Just like DOR, the collector 99.9% of the time is going on circumstantial evidence for cause of death. We rarely see it happen. Bird is dead at base of window which has a splat mark of matching feathers = "window strike" (or Frog is smushed on road with tire tracks = "dead on road" but let's not bring that one up now!)

Point is this is notable in the collection and leaving it in remarks makes is difficult to find. We can come up with all sorts of causes for blunt force trauma resulting in a dead bird (we cataloged once a hummingbird stuck in a truck grill...) but let's go with a sadly very common one. 

This really highlights the need for CT metadata description in my mind. Please let me know any of that is the issue or off the mark!

**Comment by dustymc** [comment]
@mkoo yes, that missing check from me is "I'm not sure this is quite right..." and (probably, I'm voting against myself here, not sure how truly objective I can be!) does not need to stop proceeding. Maybe we can change the template to clarify that somehow? (Can I have half a check??)

>"mortality caused" but "mortality condition"
>highlights the need for CT metadata description 

yes - https://github.com/ArctosDB/arctos/issues/5326 - nothing should be interpreted from terms (we're working on it!).

>leaving it in remarks makes is difficult to find

Agree, proceeding under my "soft objections" seems way better than that. (My objections would be softer if we were better able to fix old things, maybe we need some sort of admin-something directed that way.)

And from my "interpreted from terms" comment it occurs to me that maybe this is just a definition problem - change that to "common terminology for .... whacked into a building, often involving a window..." or something of the sort and I think this will all make sense in my head.

**Comment by Jegelewicz** [comment]
As this is the first term in a new code table, I'd like to revise the definition so we set a good example.

This will be controlled vocabulary for [mortality cause](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#mortality_cause) - **The method by which the cataloged organism died.**

The proposed definition is

> an animal that dies through impact with a window; most likely birds that cannot see the glass due to reflection of sky and foliage

I don't think that is describing a method. Instead I suggest

**Mortality due to impact with a window, this is a known issue for birds that cannot see the glass due to reflection of sky and foliage.**

**Comment by mkoo** [comment]
yes the first term of the CT should be a good one!
I would amend [mortality cause](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#mortality_cause) to : 
```The means by which resulted in or contributed to the death of the cataloged organism, and may include the circumstances of the act that produced any such injuries.```

btw That's close to the WHO def for cause of mortality too (eg https://www.who.int/data/gho/indicator-metadata-registry/imr-details/3704)

Window strike = An animal that dies through impact with a window or other reflective surface, common in birds that cannot see the glass due to reflection of sky and foliage.

better, Mr. Full checkmark?

**Comment by dustymc** [comment]
>impact with a window

I know that's what it's called, but my "objection" is that that's not all of what happens. Birds bounce off of all sorts of building-bits for all sorts of reasons, and I think that's all being lumped into this concept, whatever we call it. If that's true then I'd like a more-encompassing definition. If this is truly limited to windows then - uhh, well, I guess I just don't believe that's possible to know.

>other reflective surface

Better, but I've watched a couple die on my very non-reflective beam (probably while dodging the window I saw it happen through, but they still didn't whack glass...).

>amend

https://github.com/ArctosDB/arctos/issues/6115 - now's the time, before this gets used!

**Comment by ewommack** [comment]
> and I think that's all being lumped into this concept, whatever we call it. If that's true then I'd like a more-encompassing definition

I think they lump it all in for most of the data as well. I would be ok with it. Most of the time, they are called window strikes, and the assumption in the bird hit some part of the building, most likely the window. And the other bits of the building may direct the bird towards the window: https://www.jstor.org/stable/4162790

**Comment by Jegelewicz** [comment]
> I would amend [mortality cause](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#mortality_cause) to :
The means by which resulted in or contributed to the death of the cataloged organism, and may include the circumstances of the act that produced any such injuries.

I think that should be a separate code table request

**Comment by Jegelewicz** [comment]
> Window strike = An animal that dies through impact with a window or other reflective surface, common in birds that cannot see the glass due to reflection of sky and foliage.

Why do we feel compelled to say "an animal that dies"? We aren't describing the bird, we are describing what happened to it. Calling a bird a "window strike" is nounifying the term, which should be an adjective. (I am not usually so grammarly, but this one is bugging me).

**Mortality due to impact with a window or other reflective surface, common in birds that cannot see the glass due to reflection of sky and foliage.**

> Better, but I've watched a couple die on my very non-reflective beam (probably while dodging the window I saw it happen through, but they still didn't whack glass...).

**Mortality due to impact with a window or other man-made structure, common in birds that cannot see glass due to reflection of sky and foliage.**

When someone shows up and wants building strike, we can talk about changing this term to that and moving on or if we need both, what things are currently labeled as window strike that should be building strike.
